### PR TITLE
Improvement/tx status notification

### LINKED ISSUE #1414: Update transaction indicator
Two usability issues have come up with regards to the transaction status indicator.

### **Issue 1** 
After submitting a transaction, a status indicator appears at the bottom of the screen. If tapped, the full transaction history view is loaded moving the user away from the context the user was previously on which could be a dapp where they might be in the middle of a flow.

**Solution**: Instead of moving the user away from the context if they tap on the tx status indicator, we could instead use a **modal** to show the transaction status which keeps the user in-context and prevents them from having to navigate back to the browser.

### **Issue 2** 
The transaction indicator is not dismissable and overlaps important controls like the browsers nav bar that the user might want to access while waiting for the transaction to be processed (see screenshot below).

**Solution**: make the tx status indicator dismissible with and float above the navigation bar when in the browser

### PR DESCRIPTION
**Pull request by estebanmino**
**Description**

This PR updates the way we handle notifications
https://recordit.co/jG8hBOsEk1
and transaction details
https://recordit.co/PxKqRVBPcO

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #1414

### CODE REVIEW

File: app/core/TransactionsNotificationManager.js [hunk 1]
```diff
@@ -58,8 +59,8 @@ class TransactionsNotificationManager {
 			let message = '';
 			let nonce = '';
 
-			if (data && data.message && data.message.transaction && data.message.transaction.nonce) {
-				nonce = data.message.transaction.nonce;
+			if (data && data && data.transaction && data.transaction.nonce) {
```
Comment (andrepimenta) [inline app/core/TransactionsNotificationManager.js:62]: `data` appears twice

**Review by andrepimenta — APPROVED** [review]
Just found little typo, after that I think it can be merged.
All the changes make sense to me!  👌

File: app/core/TransactionsNotificationManager.js (hunk 1 above)
Comment (estebanmino) [reply app/core/TransactionsNotificationManager.js]: 😅

**Comment by estebanmino** [comment]
fixed issues and added this new behavior https://recordit.co/PFVoNwELc0 waiting for confirmation from @cjeria to start the review, I added some new components that need to be reviewed again

**Comment by estebanmino** [comment]
@ibrahimtaveras00 this is ready to qa again

**Review by ibrahimtaveras00 — APPROVED** [review]
All fixes look good on both OS's, QA passed 👍

**Pull request merged**